source/common/upstream/load_balancer_impl.h

```diff
@@ -710,7 +710,8 @@ class LeastRequestLoadBalancer : public EdfLoadBalancerBase {
             least_request_config.has_active_request_bias()
                 ? absl::optional<Runtime::Double>(
                       {least_request_config.active_request_bias(), runtime})
-                : absl::nullopt) {
+                : absl::nullopt),
+        selection_method_(least_request_config.selection_method()) {
```

Comment:
Does this need to check that the selection method from the config is valid, and then return POWER_OF_N_CHOICES if not?

Something like:
```
LeastRequest_SelectionMethod_IsValid(least_request_config.selection_method()) ?
    least_request_config.selection_method() :
    LeastRequest_SelectionMethod_POWER_OF_N_CHOICES
```

Reply:
I'm pretty sure it'll just default to whatever the default enum value is if unset. I would just write a quick unit test to verify/enforce that behavior.

Reply:
Acknowledging I still need to do this (didn't include in most recent commit)

Reply:
Done in `TEST_P(LeastRequestLoadBalancerTest, DefaultSelectionMethod)`